src/rules/memberOrderingRule.ts

```diff
@@ -78,12 +78,16 @@ export class MemberOrderingWalker extends Lint.RuleWalker {
     }
 
     public visitPropertyDeclaration(node: ts.PropertyDeclaration) {
-        this.checkModifiersAndSetPrevious(node, getModifiers(false, node.modifiers));
+        const { initializer } = node;
+        const isFunction = initializer != null
+            && (initializer.kind === ts.SyntaxKind.ArrowFunction || initializer.kind === ts.SyntaxKind.FunctionExpression);
+        this.checkModifiersAndSetPrevious(node, getModifiers(isFunction, node.modifiers));
         super.visitPropertyDeclaration(node);
     }
 
-    public visitPropertySignature(node: ts.Node) {
-        this.checkModifiersAndSetPrevious(node, getModifiers(false, node.modifiers));
+    public visitPropertySignature(node: ts.PropertyDeclaration) {
```

Comment:
It's debatable if `ts.PropertyDeclaration` is the right type here. However, this is what we use in other places for this method signature so I went with it here.